index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="style.css" rel="stylesheet">
+    <title>Portfolio setup and mobile</title>
+</head>
+<body>
+    <header class="toolbar">
+        <a href="#" ><p>My logo </p></a>
+        <div class ="vector">
+            <a href="#" >   <img src = "./Icons/Union.png"> 
+            </a>
+        </div>
+          
+    </header>
+      <div class="background">
+       <section class="headline">
+        <div class="header"><h1>I’m Mariane Glad to see you!</h1></div>
+        <div class="info"><h3>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h3></div>
+        <div class="connect"><h3>Lets connect</h3></div>
```

Comment:
- [ ] As requested by the previous reviewer please replace the `h3` tag with the `p` tag as this is the paragraph of the page.